dragon/numeric.rb

```diff
@@ -451,8 +451,8 @@ def map
 S
     end
 
-    self.to_i.times.map do
-      yield
+    self.to_i.times.map do |i|
```

Comment:
I think this is a breaking change.

Reply:
I believe it isn't? I don't remember what I got when I wrote it, would have to check again

Reply:
It could be due to lambda semantics :|
But relatively easy to fix if noted.. and passing real lambdas as blocks is usually rare, more advanced usage

https://docs.ruby-lang.org/en/2.7.0/Proc.html#class-Proc-label-Lambda+and+non-lambda+semantics